apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -0,0 +1,705 @@
+                              >
+                                <i class="fa fa-edit" aria-hidden="true"></i>
+                              </button>
+                            `
+                          : null}
+                      </td>
+                      <td class="align-middle text-center text-nowrap">
+                        ${renderEjs(__filename, "<%- include('../partials/scorebar'); %>", {
+                          score: instance_question.score_perc,
+                        })}
+                      </td>
+                      <td class="align-middle" style="width: 1em;">
+                        ${resLocals.authz_data.has_course_instance_permission_edit
+                          ? html`
+                              <button
+                                type="button"
+                                class="btn btn-xs btn-secondary editQuestionScorePercButton"
+                                id="editQuestionScorePerc${i_instance_question}"
+                                data-toggle="popover"
+                                data-container="body"
+                                data-html="true"
+                                data-placement="auto"
+                                title="Change question ${instance_question.question_number} percentage score"
+                                data-content="${renderEjs(
+                                  __filename,
+                                  "<%= include('../partials/editQuestionScorePercForm');%>",
+                                  {
+                                    ...resLocals,
+                                    id: 'editQuestionScorePerc' + i_instance_question,
+                                    instance_question: instance_question,
+                                  },
+                                )}"
+                                data-trigger="manual"
+                                onclick="$(this).popover('show')"
+                              >
+                                <i class="fa fa-edit" aria-hidden="true"></i>
+                              </button>
+                            `
+                          : null}
+                      </td>
+                      <td class="align-middle text-nowrap" style="width: 1em;">
+                        ${resLocals.authz_data.has_course_instance_permission_edit &&
+                        instance_question.status !== 'unanswered'
+                          ? html`
+                              <a
+                                class="btn btn-xs btn-secondary"
+                                href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                                  .id}/manual_grading/instance_question/${instance_question.id}"
+                                >Manual grading</a
+                              >
+                            `
+                          : null}
+                      </td>
+                    </tr>`;
```

Comment:
Nit (goes along with previous suggestion):

```suggestion
                    </tr>
                    `;
```

Reply:
Updated formatting locally in [6dfa82c](https://github.com/PrairieLearn/PrairieLearn/pull/9277/commits/6dfa82c90cab62f1a0b4089fc59ee6da989382de).